src/help/panel-too-small.md

```diff
@@ -0,0 +1,23 @@
+# Panel Too Small
```

Comment:
This page needs some breath:

<img width="759" height="546" alt="Image" src="https://github.com/user-attachments/assets/50e1c97b-33a5-4076-a4dd-62eecc5e7000" />

Could really benefit from more spacing between headers.

The text itself should start with *Why*.